write-fonts/src/graph/splitting.rs

```diff
@@ -279,12 +280,10 @@ fn split_pair_pos_format_2(graph: &mut Graph, subtable: ObjectId) -> Option<Vec<
         }
 
         accumulated += accumulated_delta;
-        let _largest_obj = coverage_size.max(class_def_1_size).max(class_def2_size);
-        let total = accumulated + coverage_size + class_def_1_size + class_def2_size;
-        //NOTE: harfbuzz has this optimization but I was seeing an overflow with
-        //the last object in some cases, so currently disabled
+        let largest_obj = coverage_size.max(class_def_1_size).max(class_def2_size);
+        let total = accumulated + coverage_size + class_def_1_size + class_def2_size
         // largest obj packs last and can overflow
```

Comment:
I think explaining why, such as because we only need to be able to point to the start and never the end, as well